diceRoll/diceRoll2.js

```diff
@@ -158,104 +164,91 @@ function App(context) {
     },
     addRollBarListeners: function() {
       this.context.attach('clrBtn', 'click', this.clearDisplay.bind(this));
-      this.context.attach('rollBtn', 'click', this.roll.bind(this, this.context.sumArray));
-      this.context.attach('savedBtn', 'click', this.toggleSaved.bind(this));
-    },
-    addSaveItemListeners: function() {
-      var saved = JSON.parse(this.context.storage().saved);
-      var saved_props = (Object.getOwnPropertyNames(saved));
-      for (var i = 0; i < saved_props.length; i++) {
-        var prop = saved[saved_props[i]];
-        var id = prop.id;
-        var rollArray = prop.rollArray;
-
```

Comment:
These were refactored into a their own function, while the above became its own, further down